bash_completions.go

```diff
@@ -104,31 +104,31 @@ __%[1]s_handle_reply()
     # check if we are handling a flag with special work handling
     local index
     __%[1]s_index_of_word "${prev}" "${flags_with_completion[@]}"
-    if [[ ${index} -ge 0 ]]; then
-        ${flags_completion[${index}]}
+    if [[ "${index}" -ge 0 ]]; then
+        "${flags_completion["${index}"]}"
         return
     fi
 
     # we are parsing a flag and don't have a special handler, no completion
-    if [[ ${cur} != "${words[cword]}" ]]; then
+    if [[ "${cur}" != "${words[cword]}" ]]; then
         return
     fi
 
     local completions
     completions=("${commands[@]}")
-    if [[ ${#must_have_one_noun[@]} -ne 0 ]]; then
+    if [[ "${#must_have_one_noun[@]}" -ne 0 ]]; then
```

Comment:
For my own knowledge, do `${#` really need quoting?

Reply:
I don't think so, but it doesn't hurt to have them.